packages/xforms-engine/src/parse/expression/TextChunkExpression.ts

```diff
@@ -15,19 +14,20 @@ const isOutputElement = (element: Element): element is OutputElement => {
 	return element.localName === 'output' && element.hasAttribute('value');
 };
 
-export class TextChunkExpression extends DependentExpression<'string'> {
+export class TextChunkExpression<T extends 'nodes' | 'string'> extends DependentExpression<T> {
```

Comment:
Once we add all the support for output, markdown, HTML styling, etc., this could naturally drop the `string` and only be `TextChunkExpression<'nodes'> extend DependentExpression {`. I need to experiment with those scenarios to confirm.